apps/web/app/api/mutations/admin/useChangeOverdueCourseNotification.ts

```diff
@@ -0,0 +1,32 @@
+import { useMutation } from "@tanstack/react-query";
+import { useTranslation } from "react-i18next";
+
+import { ApiClient } from "~/api/api-client";
+import { userSettingsQueryOptions } from "~/api/queries/useUserSettings";
+import { queryClient } from "~/api/queryClient";
+import { useToast } from "~/components/ui/use-toast";
+
+export function useChangeOverdueCourseNotification() {
```

Comment:
would be cool to have convention for admin endpoints to be easily indicated on the frontend also like

useAdminChange.....
ADMIN_useChange....

you can pick something with @its-gabo